src/libmysqlclient-5-macro-native-win32.patch

```diff
@@ -0,0 +1,131 @@
+This file is part of MXE.
+See index.html for further information.
+
+From 0d185ed2b345e57a5b07e7975650f7cef3977f27 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Mon, 7 Sep 2015 23:04:23 +0100
+Subject: [PATCH 5/5] macro NATIVE_WIN32 for stuff we don't have here
+
+Macro NATIVE_WIN32 is undefined in MXE. It is used instead of _WIN32
+in #ifdef's where working variant is Unix and not Win32:
+
```

Comment:
Would `_MSC_VER` be a better choice? It's likely to be suitable for upstream submission if so.